shell/platform/darwin/ios/BUILD.gn

```diff
@@ -139,13 +139,18 @@ source_set("flutter_framework_source") {
   public_configs = [ "//flutter:config" ]
 
   libs = [
```

Comment:
i'm surprised that out GN format presubmit doesn't account for these.

Reply:
Same, that formatter is usually pretty strict!